BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -322,8 +322,22 @@ open class SBAScheduledActivityManager: NSObject, SBASharedInfoController, ORKTa
         return activities.find({ $0.taskIdentifier == taskIdentifier })
     }
     
+    /**
+     Delete the output directory.
+     */
+    @objc(deleteOutputDirectoryForTaskViewController:)
+    open func deleteOutputDirectory(for taskViewController: ORKTaskViewController) {
+        guard let outputDirectory = taskViewController.outputDirectory else {return}
+        do {
+            try FileManager.default.removeItem(at: outputDirectory)
+        } catch let error as NSError {
+            debugPrint("Unable to remove ResearchKit output directory at \(outputDirectory): \(error)")
```

Comment:
What about splitting into two lines: if debug print the directory, if not then just print the error reason?